src/app/OperationalSessionSetup.cpp

```diff
@@ -320,9 +305,12 @@ CHIP_ERROR OperationalSessionSetup::EstablishConnection(const ResolveResult & re
         // is set up for LIT ICDs only when they are active, it is known that
         // the ICD is in active mode. Therefore, using mActiveRetransTimeout
         // for the initial message in the exchange is appropriate. If
-        // mActiveRetransTimeout is insufficient, consider increasing it further
-        // using sAdditionalLitBackoffInterval.
-        config.mIdleRetransTimeout = config.mActiveRetransTimeout + sAdditionalLitBackoffInterval;
+        // mActiveRetransTimeout is insufficient, consider setting it further
+        // using minimumLitBackoffInterval so that it is guaranteed to be larger
+        // than minimum lit backoff set by client and don't need worry about how larger or smaller we need to compensate.
```

Comment:
I can't make sense of this comment.  Please run it through an AI model of your choice until it makes sense.

Reply:
updated, thanks